src/js/pass/deterministic.ts

```diff
@@ -154,6 +155,21 @@ export function deterministic(rom: Rom, flags: FlagSet): void {
 
   if (flags.hardcoreMode()) hardcoreMode(rom);
 
+  useNewStatusBarGraphics(rom);
+
+}
+
+// Updates a few itemuse and trigger actions in light of consolidation
+// around item granting.
+function useNewStatusBarGraphics(rom: Rom): void {
+  const page = 0x38 << 6
+  rom.patterns[page | 0x0].pixels = Patterns.chr_page38_tile0;
```

Comment:
It would be nice to have a setter for this:

```
rom.patterns.set(0x38, 0, newTileData.HUD_LF);
rom.patterns.set(0x38, 1, newTileData.HUD_PW);
```

though I assume it's currently just a simple array so I won't ask you to change it into a fancy subclass.  We could do `rom.setChrTile(0x38, 0, HUD_LF)` easily enough as well.